core/tests/webdriverio_utils/BlogDashboardPage.js

```diff
@@ -148,6 +152,20 @@ var BlogDashboardPage = function() {
     await waitFor.invisibilityOfSuccessToast('Blog Post Saved Successfully.');
   };
 
+  this.updateAuthorDetails = async function(
+      authorName, authorBio
+  ) {
```

Comment:
This doesn't fit all on one line? It looks like it would

Reply:
Done